virttest/remote_commander/remote_runner.py

```diff
@@ -768,19 +791,31 @@ def remote_agent(in_stream_cls, out_stream_cls):
     :params out_stream_cls: Class encapsulated output stream.
     """
     try:
-        fd_stdout = sys.stdout.fileno()
-        fd_stderr = sys.stderr.fileno()
-        fd_stdin = sys.stdin.fileno()
+        sys.stdout.flush()
+        sys.stderr.flush()
+        fd_stdin = sys.stdin.fileno()    # == 0
+        fd_stdout = sys.stdout.fileno()  # == 1
+        fd_stderr = sys.stderr.fileno()  # == 2
+        orig_stdout = os.dup(fd_stdout)  # Original stdout to master.
+        orig_stderr = os.dup(fd_stderr)  # Original srderr to master.
```

Comment:
Seems this orig_stderr not used
I'm not familiar  with remote runner, so just check this patch code here, no more comment from me.